test/Plutip.purs

```diff
@@ -146,9 +146,7 @@ import Wallet.Cip30Mock
 main :: Effect Unit
 main = launchAff_ do
   Utils.interpretWithConfig
-    -- we don't want to exit because we need to clean up after failure by
-    -- timeout
-    defaultConfig { timeout = Just $ wrap 50_000.0, exit = false }
+    defaultConfig { timeout = Just $ wrap 50_000.0, exit = true }
```

Comment:
I think it got set back to false because of the misleading comment. It's the case (today at least) that the test runner only exists on all tests being run, not the first failure.

Reply:
Could you please check that when one of the tests fails by timeout, the services are killed and the subsequent tests can run

Reply:
I checked with a 5 second timeout, which is enough for processes to start being launched, and the following tests continue to hit their timeout, implying they weren't immediately failing due to port conflicts.

I'll try and find a value in which some do pass and some timeout

Reply:
Check with 15 sec to be sure the action gets cancelled in Contract code

Reply:
```
  ✓︎ runPlutipContract
  ✓︎ runPlutipContract: Pkh2Pkh
  ✗ runPlutipContract: Pkh2Pkh with stake key:

  test timed out after 20000ms
  ✓︎ runPlutipContract: parallel Pkh2Pkh
  ✗ runPlutipContract: parallel Pkh2Pkh with stake keys:

  test timed out after 20000ms
```

Looks good to me

Reply:
This can be seen actually in this PRs currently running check:

https://ci.staging.mlabs.city/build/12715/nixlog/1

A bit concerning that some are still hitting the 50s timeout

Reply:
No longer the case, I guess it's dependent on CI load